chain/client/src/client.rs

```diff
@@ -71,6 +73,18 @@ pub struct ClientActor {
     num_tx_processed: u64,
 }
 
+fn wait_until_genesis(genesis_time: &DateTime<Utc>) {
+   let now : DateTime<Utc> = Utc::now();
+   //get chrono::Duration::num_seconds() by deducting genesis_time from now
+   let chrono_seconds: i64 = now.signed_duration_since(*genesis_time).num_seconds();
+   //check if number of seconds in chrono::Duration larger than zero
+   if chrono_seconds > 0  {
+       info!(target: "chain", "Waiting {} seconds until genesis ", chrono_seconds);
```

Comment:
extra space at the end?
also I think it will be more informative `"Waiting until genesis: {}", genesis_time` because something like `269568 seconds` won't be super clear :)

Reply:
I updated this logging by "Waiting until genesis: {}", chrono_seconds cause we have to wait for certain amount of seconds, does it sound good?

Reply:
Well, I meant to show until when we are waiting instead of how long we are waiting.
But let's merge this now.